happytransformer/tc/default_args.py

```diff
@@ -1,27 +1,32 @@
 ARGS_TC_TRAIN = {
     'learning_rate': 5e-5,
+    'num_train_epochs': 3.0,
     'weight_decay': 0,
     'adam_beta1': 0.9,
     'adam_beta2': 0.999,
     'adam_epsilon': 1e-8,
     'max_grad_norm':  1.0,
-    'num_train_epochs': 3.0,
+    'save_preprocessed_data': False,
+    'save_preprocessed_data_path': "",
+    'load_preprocessed_data': False,
+    'load_preprocessed_data_path': "",
+}
+
+# as of right now we do not provide custom settings
+ARGS_TC_EVAL = {
+    'save_preprocessed_data': False,
```

Comment:
we totally do provide custom setting though? Users can override `args` in various methods

Reply:
0b18230

Reply:
Fixed